mailscanner/conf.php.example

```diff
@@ -39,6 +39,11 @@ define('DEBUG', false);
 // Define language (de, en, fr, it, nl, pt_br);
 define('LANG', 'en');
 
+//Session Handling - conflicts can exist when the your environment makes use of multiple php sessions on the same server
+// to resolve this, uncomment the following option.  See https://github.com/mailwatch/MailWatch/issues/730 for more info
+//define('SESSION_NAME', 'MailWatch');
+
+
```

Comment:
Double blank line to remove.

`// Session Handling` missing space after //

Not sure, it really needed to ref to #730.